common-text-blocks.md

```diff
@@ -0,0 +1,35 @@
+
+# Reusable reference blocks
+
+This section is for reusable blocks to establish standards for presentation
+within our Documentation in regards to our blocks of warnings and other
+common areas. These use Vuepress' slots for danger, warning, tip and details.
+Use the first two as sparingly as possible unless ***ABSOLUTELY*** necessary.
+
+
+## For sections that are under construction.
+
+``` ::: warning Under Construction
+
+The documentation here is under heavy construction. Updates are coming, and
+likely in the works. If not, please report a issue in our repository for this
+site. You can find on the page linked in our header.
+
+:::
```

Comment:
```suggestion
``` ::: warning Under Construction

This document is under construction, please check back soon for updates.
Please see [our socials](link/to/socials/list) and feel free to ask for assistance or enquire as to the status of this document.

:::
```

I think asking for GH issues to be raised for WIP documents might be soliciting unnecessary issues to be created. If it is being actively worked on then I think it better to come to Discord etc. and ask.
Once there then various people can help answer the question if the info is currently missing from the doc and has the side effect of drawing attention to the doc if it is currently stuck behind a blocker or has stalled.